compiler/loco/src/Service/TypeInference.test.cpp

```diff
@@ -257,23 +260,29 @@ TEST(MultiDialectTypeInferenceRuleTest, test1)
   loco::link(graph_output, push_node);
 
   // initially they don't have type info
+  ASSERT_FALSE(loco::dtype_known(s4_node));
   ASSERT_FALSE(loco::dtype_known(s8_node));
   ASSERT_FALSE(loco::dtype_known(u8_node));
 
   // Run Type Inference
   TestTypeInferenceRule<loco::DataType::U8> u8_rule;
   TestTypeInferenceRule<loco::DataType::S8> s8_rule;
+  TestTypeInferenceRule<loco::DataType::S4> s4_rule;
   loco::CanonicalTypeInferenceRule canon_rule;
 
   loco::MultiDialectTypeInferenceRule rules;
 
   rules.bind(TestDialect<loco::DataType::S8>::get(), &s8_rule)
     .bind(TestDialect<loco::DataType::U8>::get(), &u8_rule)
+    .bind(TestDialect<loco::DataType::S4>::get(), &s4_rule)
     .bind(loco::CanonicalDialect::get(), &canon_rule);
```

Comment:
Don't need to consider the bind order to fit the graph node order?

Reply:
ah... I think it should work without any orders.
I just added without noticing orders like in s4 - s8 - u8...
something in mind with "just to make less changes"